Livecheckables/mesa.rb

```diff
@@ -0,0 +1,6 @@
+class Mesa
+  livecheck do
+    url "https://mesa.freedesktop.org/archive/"
```

Comment:
```suggestion
    url "https://archive.mesa3d.org"
```

The above URL redirects to https://archive.mesa3d.org/, so we should use that to avoid the redirection.